src/avro/src/reader.rs

```diff
@@ -258,13 +258,9 @@ impl<R: AvroRead> Reader<R> {
     fn fill_buf(&mut self, n: usize) -> Result<(), AvroError> {
         // We don't have enough space in the buffer, need to grow it.
         if n >= self.buf.capacity() {
```

Comment:
I think this needs to be `len` now, not `capacity`.

Reply:
you're right, fixed